content/yaml-quick-start/white-label-getting-started.md

```diff
@@ -0,0 +1,463 @@
+
+To trigger a build using the Codemagic REST API, you need your API access token, the application id, and the workflow id. 
+
+- The access token is available in the Codemagic UI under **Teams > Personal Account > Integrations > Codemagic API > Show**. You can then store this as an environment variable if you are calling the REST API from other workflows.
+- Once you have added your app in Codemagic, open its settings and copy the **application id** from the browser address bar - `https://codemagic.io/app/<APP_ID>/settings`
+- The workflow id is the string value you assigned to the `name` property e.g "ios-qa-build"
+
+An example of triggering a single build and passing an environment variable to specify the client id and a group to read the variables from might look like this:
+
+
+{{< highlight yaml "style=paraiso-dark">}}
+- name: Trigger single client builds
+  script: | 
+    CLIENT="001"
+    curl -H "Content-Type: application/json" -H "x-auth-token: ${CM_API_KEY}" \
+      --data '{
+          "appId": "<app-id>", 
+          "workflowId": "<workflow-id>",
+          "branch": "<branch-name>",
+          "labels": ["'${CLIENT}'"],
+          "environment": { 
+              "variables": { 
+                  "CLIENT_ID": "'${CLIENT}'"
+              },
+              "groups": [
+                  "WL_${CLIENT}"
+              ]
+          }
+        }' \
+       https://api.codemagic.io/builds
+{{< /highlight >}}
+
+
+
+In the following example, to trigger builds for clients `001`, `002` and `003` a simple array is first defined and then a for loop is used to initiate a build for each element in the array. The unique `CLIENT_ID` variable is provided in the payload for the three builds that are started when this command is run.
+
+{{< highlight yaml "style=paraiso-dark">}}
+- name: Trigger multiple client builds
+  script: | 
+      CLIENTS=("001" "002" "003")
+      for CLIENT in ${CLIENTS[@]}; do
+          echo "CLIENT: $CLIENT"  
+          curl -H "Content-Type: application/json" -H "x-auth-token: ${CM_API_KEY}" \
+              --data '{
+                  "appId": "<app-id>", 
+                  "workflowId": "<workflow-id>",
+                  "branch": "<branch-name>",
+                  "environment": { 
+                      "variables": { 
+                          "CLIENT_ID": "'${CLIENT}'"
+                      },
+                      "groups": [
+                          "WL_${CLIENT}"
+                      ]                      
+                  }
+              }' \
+          https://api.codemagic.io/builds
+      done
+{{< /highlight >}}
```

Comment:
There's no mention of how to include this in the workflow so if you merge from one branch to another it will trigger all the builds. It sounds like you can only start the builds by manually running this script yourself.